### PR TITLE
move duplicated code to annonymous namespace static function

### PR DESCRIPTION
**Pull request by narunlifescience**

### CODE REVIEW

**Comment by hatstand** [comment]
Just put it all in a static function in an anonymous namespace in the .cpp file.

**Comment by narunlifescience** [comment]
Done! now we wont be able to use const though..

File: src/widgets/fancytabwidget.cpp [hunk 1]
```diff
@@ -55,6 +55,38 @@ using namespace Internal;
 const int FancyTabBar::m_rounding = 22;
 const int FancyTabBar::m_textPadding = 4;
 
+
+namespace {
+static void selectTab(QPainter *&p, const QRect rect) {
```
Comment (hatstand) [inline src/widgets/fancytabwidget.cpp]: `void SelectTab(QPainter* p, const QRect& rect) {`

**Comment by narunlifescience** [comment]
Done!

**Pull request merged**